content/blog/2025-03-17-hard-fork-1.md

```diff
@@ -0,0 +1,47 @@
+
+To continue the positive momentum of Neptune, we have decided to increase the block size limit sooner rather than later. The exact changes can be read in [this pull request](https://github.com/Neptune-Crypto/neptune-core/pull/502).
+
+Those of you who are only interested in the mechanics of Neptune can stop reading here. But if you want some insight into the philosophy guiding our work, you may find the rest of the article interesting.
+
+## The good, the bad, and the ugly
+
+There are three relevant definitions when considering changes to a blockchain: soft forks, hard forks, and reorganizations. A soft fork is a change to the rules where old versions of the software still consider the new blocks valid. If we for example were to make a rule that all block hashes had to start with the hexadecimal character "a", that would be a soft fork, as the old neptune-core versions would still consider the new blocks valid. A hard fork is a change to the rules such that new blocks will be considered invalid by the old software, an increase in the maximum allowed block size is a good example of this.  And a reorganization is the rewriting of history: Where the state of the blockchain is rolled back to an earlier state thus deleting some transactions from history.
+
+In an idealized world, the rules of a blockchain would never change, so neither soft forks, hard forks, or reorganizations would be needed. But since development takes time and we all need feedback from the real world, not all changes can be considered equally bad. We consider soft forks the least problematic and reorganizations the most problematic[^5]. With hard forks in the middle.
+
+It's worth noting though that we consider hard forks problematic for two reasons:
+1. They risk splitting the network thus creating confusion about the true state of the blockchain
+2. They introduce an element of arbitrary rule which is antithetical to the ethos of Blockchain: money without rulers.
+
+To provide a level of expectation and restrict the arbitrary power of current and future Neptune developers, the following list shows which hard forks we foresee:
+1. increase in block size limit from 2MB to 8MB.
+2. Better representation of transaction inputs, increasing transaction throughput by at least 10x.
```

Comment:
Link to the fast authentication structure authentication article again, or remind the reader that this corresponds to the long-term solution described earlier.
```suggestion
2. Better (irredundant) representation of transaction inputs, increasing transaction throughput by at least 10x.
```